config/services.yml

```diff
@@ -23,6 +23,32 @@ x-turbo-config: &default-turbo-config
 
 https://cmr.earthdata.nasa.gov:
 
+  - name: harmony/download
+    description: Service that performs no processing, it only returns download links for all of the matched granules from the CMR.
+    data_operation_version: '0.18.0'
```

Comment:
This defines the JSON passed to each backend service in the chain (in this case just query-cmr). I'm assuming 0.18.0 is the latest version since that's what all the other service chains currently define.